packages/matchbox/src/components/Tooltip/Tooltip.tsx

```diff
@@ -151,59 +196,4 @@ function Tooltip(props) {
 }
 
 Tooltip.displayName = 'Tooltip';
-Tooltip.propTypes = {
-  as: PropTypes.string,
-  id: PropTypes.string.isRequired,
-  content: PropTypes.node,
-  /**
-   * Disables hover events
-   */
-  disabled: PropTypes.bool,
-  dark: deprecate(PropTypes.bool, 'Use system props to set styles', 'error'),
-  active: PropTypes.bool,
-  left: PropTypes.bool,
-  right: PropTypes.bool,
-  top: PropTypes.bool,
-  bottom: PropTypes.bool,
-  horizontalOffset: PropTypes.string, // TODO Either deprecate or style Tooltip tips
-  /**
-   * Disables automatic positioning
-   */
-  forcePosition: PropTypes.bool,
-  children: PropTypes.node,
-  /**
-   * These props are provided automatically through the Overlay component
-   */
-  preferredDirection: PropTypes.shape({
-    top: PropTypes.bool,
-    bottom: PropTypes.bool,
-    left: PropTypes.bool,
-    right: PropTypes.bool,
-  }),
-
-  /**
-   * Element ID for the portal that will house tooltips. Appends to body if not provided.
-   */
-  portalId: PropTypes.string,
-  ...createPropTypes(border.propNames),
-  ...createPropTypes(color.propNames),
-  ...createPropTypes(layout.propNames),
-  ...createPropTypes(padding.propNames),
-  ...createPropTypes(typography.propNames),
-  ...createPropTypes(shadow.propNames),
-};
-
-Tooltip.defaultProps = {
-  right: true,
-  bottom: true,
-  horizontalOffset: '0px',
-  forcePosition: false,
-  preferredDirection: {
```

Comment:
Do we need to update these defaults in the props spread?

Reply:
these actually weren't used at all, i think we just forgot to remove them at some point